src/utils/utils_windows_intrin.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#ifndef UMF_UTILS_WINDOWS_INTRIN_H
+#define UMF_UTILS_WINDOWS_INTRIN_H 1
+
+#ifdef _WIN32
+
+// Disable warning 28251: "inconsistent annotation for function" thrown in
+// intrin.h, as we do not want to modify this file.
+#pragma warning(push)
+#pragma warning(disable : 28251)
+
+#include <intrin.h>
+
+#pragma warning(pop)
+
+#pragma intrinsic(_BitScanForward64)
+#pragma intrinsic(_BitScanReverse)
```

Comment:
hmm... now we have both these pragmas included in both files, instead of single required pragma per file - is it ok?

Reply:
I do not know. Moved to other files.
Done.